ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/EmailIntegrationForm.tsx

```diff
@@ -216,25 +243,65 @@ function EmailIntegrationForm({
                             isDisabled={!isEditable}
                         />
                     </FormLabelGroup>
+                    <FormLabelGroup
+                        label=""
+                        fieldId="notifier.email.unauthenticated"
+                        errors={errors}
+                    >
+                        <>
+                            <div className="pf-u-display-flex pf-u-align-items-flex-start">
+                                <Checkbox
+                                    label="Enable unauthenticated SMTP"
+                                    id="notifier.email.unauthenticated"
+                                    isChecked={allowUnauthenticatedSmtp}
+                                    onChange={onUpdateUnauthenticatedChange}
+                                    onBlur={handleBlur}
+                                />
+                                <ToggledPopoverElement
+                                    popoverContent="Enable unauthenticated SMTP will allow you to setup the RHACS email notifiers if you don’t have authenticated email services."
```

Comment:
Not a blocker to merge:
* Unless there is a difference in behavior between Red Hat and open-source StackRox, we omit reference to product branding.

Reply:
Good callout. We can change that but who typically signs off on copy changes?

Reply:
Good question. I will answer via direct message.

Reply:
Let's just drop the part that reads "the RHACS"; that qualification is unnecessary to the meaning.

Also, for a small change for clarity like this, I agree with Grace Hopper, "If it's a good idea, go ahead and do it. It's much easier to apologize than it is to get permission."

```suggestion
                                    popoverContent="Enable unauthenticated SMTP will allow you to setup an email notifier if you don’t have authenticated email services."
```